### PR TITLE
TM-6124-Projected-Vacancies-API

### PR DESCRIPTION
**Pull request by cagbakwu**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/3013)

[Ticket](https://metaphase.atlassian.net/browse/TM-6124)

### CODE REVIEW

**Review by M00staff — CHANGES_REQUESTED** [review]
failing tests

**Comment by elizabeth-jimenez** [comment]
These are the wrong endpoints

**Comment by cagbakwu** [comment]
@elizabeth-jimenez 
> These are the wrong endpoints

That's my bad! I assumed because the names were the same...Ill resolve it now.

File: talentmap_api/fsbid/views/admin_projected_vacancies.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class FSBidAdminProjectedVacancyFiltersView(APIView):
 
     # ======================== Get PV Filters ========================
 
-    permission_classes = [IsAuthenticated, Or(isDjangoGroupMember('bureau_user'), isDjangoGroupMember('superuser'), ) ]
```
Comment (cagbakwu) [inline talentmap_api/fsbid/views/admin_projected_vacancies.py:22]: Only required for the READ requests

**Comment by cagbakwu** [comment]
Just realized I used  projected_vacancies instead of admin_projected_vacancies 🤦🏾‍♂️

**Comment by M00staff** [comment]
this looks right - did you double check & log in as an AO to make sure everything works?

sometimes you need to log out and log in twice locally to get the permissions to change

**Comment by M00staff** [comment]
still a WiP?

image

**Review by M00staff — APPROVED** [review]

**Pull request merged**